src/TextView.js

```diff
@@ -0,0 +1,155 @@
+import * as Notification from './utils/Notification.js';
+
+/** @module TextView */
+
+const $ = require('jquery');
+
+/*
+ * Class that manages getting the text for syllables in Neon from the mei file
+ */
+class TextView {
+	/**
+	 * A constructor for a TextView.
+	 * @param {NeonView} neonView = The NeonView parent.
+	 */
+	constructor(neonView) {
+		this.neonView = neonView;
+		let notificationSent = false;
```

Comment:
Why is this not instance specific (`this.notificationSent`)? Do we need to reference this from a static context?